lib/llm/providers/gemini.rb

```diff
@@ -2,26 +2,26 @@
 
 module LLM
   require "net/http"
-  require "uri"
   require "json"
   require "llm/http/client"
   require "llm/adapter"
   require "llm/message"
 
   class Gemini < Adapter
-    BASE_URL = "https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash"
+    HOST = "generativelanguage.googleapis.com"
+    PORT = 443
+    PATH = "/v1beta/models/gemini-1.5-flash"
```

Comment:
@antaz Looks like Gemini specify the model in the path rather than query string. We might want this to be `/v1beta/models/`, and then do something like: `[PATH, params[:model].join("/")`

Reply:
@0x1eef That's a good idea, for now we will keep the model name hardcoded until we figure out how to pass the model parameters